### DISCUSSION TITLE
Loading the same IR file give almost the same reverb but the TOOB version sounds different after 1 second..... ?

### DISCUSSION THREAD

**Discussion by maarthome** [rerdavies/pipedal#161] [Q&A]
Hi Robin,

Perhaps it is intended but the Toob Convolution Reverb (stereo) sounds compared with the x42 Convolution Stereo plugin...?
Not in the beginning, but only after 1 second.... the Toob reverb 'squeezes' the volume and returns to normal volume again... This doe not seem intended I think...

Not a big issue, but perhaps you want to know.... 

Cheers,

Maarten

(Hope you do not get the feeling I am overspamming you. But in the past I did this too little: ginving feedback)
(And... I do love the Toob stereo flanger..... 🎶)

**Comment by rerdavies** [comment]
Click on the (i) button in the plugin control panel, and read about the T control. 

If you can hear the difference, you probably need to increase the T control.  It works for some reverb impulse files, but not always for reverb impulse files with very long tails.

All questions and feedback is valuable, especially if it lead to bug fixes, or ideas for enhancements, so please keep them coming.

(And yes!  I do love the Toob Stereo Flanger too! Such an amazing difference for a ridiculously easy code tweak. :-) )
